pointstorm/embedding/text_tests.py

```diff
@@ -52,6 +52,42 @@ def test_generate_embedding_success(self, mock_print):
         # Check that embeddings are there
         self.assertGreaterEqual(len(result.embeddings), 1)
 
+class TestOpenAIEmbedding(unittest.TestCase):
```

Comment:
Test looks good, could you add example usage as well?

Reply:
Added to the bottom of this file as a comment.